moveit_ros/moveit_servo/test/launch/test_servo_collision.test.py

```diff
@@ -1,7 +1,7 @@
 import os
 import sys
 import unittest
-
+import time
```

Comment:
nitpick:  please alphabetize. Put `import time` before `import unittest`

Reply:
That's fair. Changed it.